query/keyvault/keyvault_with_rbac_key_expiration_set.sql

```diff
@@ -0,0 +1,31 @@
+with rbac_vault as (
+  select
+    name
+  from
+    azure_key_vault
+  where enable_rbac_authorization
+)
+select
+  -- Required Columns
+  kvk.id as resource,
+  case
+    when v.name is null then 'skip'
+    when enabled and expires_at is null then 'alarm'
+    else 'ok'
+  end as status,
+  vault_name || ' key ' || kvk.name ||
+    case
+      when v.name is null then kvk.vault_name || ' not RBAC enabled vault.'
```

Comment:
Similar to  https://github.com/turbot/steampipe-mod-azure-compliance/pull/112/files#r929971600